server/src/test/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileControllerTest.java

```diff
@@ -9,278 +9,278 @@
 import io.micronaut.http.client.annotation.Client;
 import io.micronaut.http.client.exceptions.HttpClientResponseException;
 import io.micronaut.http.hateoas.Resource;
-import io.micronaut.test.annotation.MicronautTest;
-import io.micronaut.test.annotation.MockBean;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import javax.inject.Inject;
 import java.time.LocalDate;
 import java.util.*;
 
-import static com.objectcomputing.checkins.services.memberprofile.MemberProfileTestUtil.*;
 import static com.objectcomputing.checkins.services.role.RoleType.Constants.MEMBER_ROLE;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.*;
 
-@MicronautTest
-public class MemberProfileControllerTest {
+import com.objectcomputing.checkins.services.TestContainersSuite;
+import com.objectcomputing.checkins.services.fixture.MemberProfileFixture;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+public class MemberProfileControllerTest extends TestContainersSuite implements MemberProfileFixture {
 
     @Inject
     @Client("/services/member-profile")
     private HttpClient client;
 
-    @Inject
-    MemberProfileServices mockMemberServices;
+    public static MemberProfileUpdateDTO mkUpdateMemberProfileDTO() {
```

Comment:
This method looks like a duplicate of MemberProfileTestUtil.mkUpdateMemberProfileDTO()

Could we remove this?

Reply:
removed and updated